tests/integration_tests/style/test_rust.py

```diff
@@ -12,10 +12,8 @@ def test_rust_style():
     @type: style
     """
 
-    #  ../src/io_uring/src/bindings.rs
-    config = open("fmt.toml", encoding="utf-8").read().replace("\n", ",")
     # Check that the output is empty.
-    _, stdout, _ = utils.run_cmd(f"cargo fmt --all -- --check --config {config}")
+    _, stdout, _ = utils.run_cmd("cargo fmt --all -- --check --config-path fmt.toml")
```

Comment:
Shouldn't this also need to be run with `+nightly`?

Reply:
This would have, but I have dropped nightly from the toolchain here. This generates warnings, but the test doesn't vet or fail for warnings.